src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Windows.cs

```diff
@@ -942,8 +942,8 @@ public static ProcessInfo[] GetProcessInfos()
 
         private static unsafe ProcessInfo[] GetProcessInfos(IntPtr dataPtr)
         {
-            // 60 is a reasonable number for processes on a normal machine.
-            Dictionary<int, ProcessInfo> processInfos = new Dictionary<int, ProcessInfo>(60);
+            // 250 is a reasonable number for processes on a normal machine.
+            var processInfos = new List<ProcessInfo>(250);
```

Comment:
> On my desktop box there are ~200 so I figured this assumption is out of date.

It may be that we should pick a better starting point.  I just don't have a good sense of what that would be, and we'd probably want to sample various machines on various OSes in various configurations to pick a good one.  If it's impactful, we could of course also make it tuned dynamically, such that we remember the count on the previous call(s) and use that to influence the starting size on subsequent ones, but that might be more engineering than this is worth.